pkg/kubectl/cmd/logs.go

```diff
@@ -309,6 +320,10 @@ func DefaultConsumeRequest(request *rest.Request, out io.Writer) error {
 	}
 	defer readCloser.Close()
 
-	_, err = io.Copy(out, readCloser)
-	return err
+	scanner := bufio.NewScanner(readCloser)
+	for scanner.Scan() {
+		fmt.Fprintf(out, "%s\n", scanner.Text())
```

Comment:
Ah, e2e tests are failing with `--limit-bytes`, because of `\n` at the end. I'll have a look later.